docs/en/framework/ui/angular/abp-window-service.md

```diff
@@ -14,9 +14,9 @@ Firstly, ensure that the service is injected into the component or any other Ang
 ```js
 import { AbpWindowService } from '@abp/ng.core';
 
-constructor(private abpWindowService: AbpWindowService) { }
+private abpWindowService   = inject(AbpWindowService) 
```

Comment:
We will already have the old way of injection in the docs, so we can just make a replacement instead

```ts
import { AbpWindowService } from '@abp/ng.core';
import { inject } from '@angular/core';

private abpWindowService = inject(AbpWindowService) 
```